### ISSUE TITLE
login does not preserve state

### ISSUE THREAD

**Issue by alexy** [scala-exercises/scala-exercises#564]
Started an exercise, hit Run, was asked to login and then dropped into homepage.  I'd rather resume Run with the input given...  A+

**Comment by dialelo** [comment]
Good point, we could store the anonymous progress in local storage and merge it when the user progress when they log in.
